extensions/parallels-virtual-machines/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Parallels Virtual Machines Changelog
 
+## [Update] - 2024-02-12
+
+- Added ability to change colorsheme of icon in the list of virtual machines via extension preferences.
```

Comment:
I feel this feature is in a bit of tension. On the one hand, I understand wanting Raycast to look the way you want it to. On the other hand, this is a lot of extra complexity for a feature that doesn't add anything to the _purpose_ of this extension.

I'm happy to accept this for now, but I wonder if the Raycast team would investigate other options, or perhaps others exist and there could be more standards around what's expected here. Could Raycast have a monochrome vs colour preference? Should extensions detect dark/light mode and use an appropriate icon set (is that possible right now?, is that the motivating factor in this change @jakucermak?).

@pernielsentikaer what do you think about ecosystem-wide ways of approaching this?

Reply:
We already have the possibility to set different image for light/dark either with [Image.Source](https://developers.raycast.com/api-reference/user-interface/icons-and-images#image.source) or with [environment.appearance](https://developers.raycast.com/api-reference/environment)

Reply:
Ah perfect! In that case I'd be in favour of using that to power this instead of a preference.